api/v1/model_handlers.go

```diff
@@ -272,6 +274,20 @@ func (a *Api) RegisterModel(container *restful.Container) {
 		Param(ws.QueryParameter("end", "End time for requested metric").DataType("string")).
 		Writes(MetricResult{}))
 
+	// The /sum/namespaces/{namespace-name}/pods/{pod-name}/metrics/{metric-name} endpoint exposes
+	// an aggregated metric for a list od pods of the model.
+	ws.Route(ws.GET("/sum/namespaces/{namespace-name}/pods/{pod-list}/metrics/{metric-name}").
```

Comment:
@mwielgus: I chatted with @afein offline and that discussion is captured here #476. The current API format is not well suited for adding more use cases. This PR is a classic example of that. 
Essentially if we had a `/api/v1.1/stats/namespaces/<namespace>/pods` to begin with which returned all the stats for pods under the given namespace, that should satisfy your use case, I hope. In the future, we can add support for label selectors as query params, which should help result in querying only a subset of pods. 
In general, +1 for doing client side summation for now. Once we understand the use cases more, we can consider moving operations to the server side. WDYT?